add curation policy (#90)
Joystream/helpdesk: roles/content-curators/README.md

The main job for curators is to regularly check the content directory and channels, and checking that rules and guidelines are being followed.

## Curation Policy
Due to the large influx of users, and incentives for video curation, we need a formal workflow for the `Content Curators`, and a curation policy.

> Note that parts of this policy only applies to this testnet.

Reviewer: ```suggestion
Note that parts of this policy only applies to this testnet (Babylon).
```

- The data in "Changes Made Since Last Report - Owner" can be found by said script.
- The data in "Changes Made Since Last Report - Curators" can be found using the same script, in addition to copy/pasting data the from the [Regular Checks](#regular-checks) - which _should_ be correct if all actions were reported there..
- "All Non-Playable Videos" and "All Non-Visible Channels" should be extractable from older reports, and data already filled in.

### Policy

`warning`
Means posting a note in the forum, as part of the [Regular Checks](#regular-checks), that something has to be corrected within a certain time period

`hidden`
Means updating the status (`isPublic`) for a video from `true` to `false`. Note that this can be changed back to `true` by the owner!

`censor`
Means updating the status (`isCensored`) for a video from `true` to `false`. Unlike `hidden`, this can not be changed back by anyone but another curator.

#### Videos
**When to (only) issue a `warning`:**
- Missing artwork
  - If the thumbnail is missing, or just really poor, the video can be `hidden`, or issued a `warning` (Council decides u.n.o.).
- Suspicious license

>   - If the Curator suspects, but is not able to ensure, a video is incorrectly licensed, a `warning` can be issued, leaving the channel owner some defined deadline to respond.

Reviewer: ```suggestion
  - If the Curator suspects, but is not able to verify that a video is incorrectly licensed, a `warning` can be issued, leaving the channel owner some defined deadline to respond and provide more information or evidence.
```

**When to make a video `hidden`:**
- The content must be as "advertised"
  - If the title, description, category and thumbnail implies a baking video, the video should not be a documentary about Bitcoin
- Missing artwork
  - If thumbnail is missing, or just really poor, the the video can be hidden, or given a warning (Council decides u.n.o.)
- Poor quality
  - If the video quality is "unreasonably" low

**When to `censor` a video:**
- License requires attribution

>   - If the selected license requires attribution, but non is given the, or attribution is incorrect

Reviewer: ```suggestion
  - If the selected license requires attribution, but none is given the, or attribution is incorrect
```

> - Suspected copyright violation

Reviewer: Perhaps add above new bullet point `Any other breach of License terms`

  - If it's not clear that the video is in violation, but this is strongly suspected by the Curator
- Status change by owner without resolving issue
  - If a video is `hidden` by a curator, and the channel owner makes it public again without correcting the issue described by the curators
- Terms of Service violation
  - If the video is in violation of other parts of the [ToS](https://testnet.joystream.org/#/pages/tos)

#### Channels
**When to (only) issue a `warning`:**
- Missing artwork

>   - If the avatar or cover is missing, or just really poor, the channel can be `hidden`, or issued a `warning` (Council decides u.n.o.).

Reviewer: ```suggestion
  - If the avatar or cover is missing, or just really poor, the channel can be `hidden`, or issued a `warning` (Council decides).
```

Reviewer: Council decides?
Shouldn't it be curators?

Author: I see your point, they have the hammer! However, the Council should be able to set overarching policies such as this.
